Misc/NEWS.d/next/Build/2017-11-02-20-30-57.bpo-28791.VaE3o8.rst

```diff
@@ -0,0 +1 @@
+Update Windows installer to use SQLite 3.21.0.
```

Comment:
I would make this "Windows builds" - the installer doesn't have anything to do with SQLite.

Reply:
Updated.